doc/developer/common/api/json_api.md

```diff
@@ -20,8 +20,6 @@ Currently the following endpoints are provided:
 | GET    | /api/event_kinds/                                 | List all accessible events kinds                                                |
 | GET    | /api/event_kinds/:id                              | Fetch a single event kind, replace :id with the event's primary key             |
 
-Visit your hitobito's swagger UI [/api-docs](/api-docs) for detailed documentation and a sandbox for testing/developing requests.
-
```

Comment:
Diese Datei wird in der Applikation unter /api angezeigt. Daher muss der Link auf die Swagger Doku dort irgendwie mit rein, idealerweise noch viel prominenter als jetzt irgendwo mitten im Dokument. Bis eine andere Lösung umgesetzt ist, würde ich diesen Link hier belassen, da diese Datei wohl viel häufiger in der Applikation als auf Github angeschaut wird.